apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -221,7 +221,7 @@ export const getTextRender = (search?: FilterValue | string) => {
     }
 }
 
-const getIntegerPartOfReading = (value: string) => value?.split(DEFAULT_CURRENCY_SEPARATOR)[0]
+const getIntegerPartOfReading = (value: string) => isString(value) && value.split(DEFAULT_CURRENCY_SEPARATOR)[1].includes('0000') ? value[0] : value
```

Comment:
it looks like a hack. Probably, you should check the meter `units of measurement`